crates/bevy_app/src/app.rs

```diff
@@ -23,12 +23,9 @@ bevy_utils::define_label!(
 );
 
 #[allow(clippy::needless_doctest_main)]
-/// A container of app logic and data.
+/// An ECS application that wraps a [`World`], a runner function, and a sub-[`App`] collection.
 ///
-/// Bundles together the necessary elements like [`World`] and [`Schedule`] to create
-/// an ECS-based application. It also stores a pointer to a [runner function](Self::set_runner).
-/// The runner is responsible for managing the application's event loop and applying the
-/// [`Schedule`] to the [`World`] to drive application logic.
+///[`App`] instances are constructed using a builder pattern.
```

Comment:
This line is useful and should still be added, but an external link to a resource on the builder pattern would be helpful.

Reply:
Good idea